ibis-server/tests/routers/ibis/test_clickhouse.py

```diff
@@ -0,0 +1,396 @@
+import base64
+import os
+
+import clickhouse_connect
+import orjson
+import pandas as pd
+import pytest
+from fastapi.testclient import TestClient
+from testcontainers.clickhouse import ClickHouseContainer
+
+from app.main import app
+
+client = TestClient(app)
+
+
+@pytest.fixture(scope="class")
+def clickhouse(request) -> ClickHouseContainer:
+    ch = ClickHouseContainer("clickhouse/clickhouse-server:head-alpine", port=8123)
+    ch.start()
+    client = clickhouse_connect.get_client(
+        host=ch.get_container_host_ip(),
+        port=int(ch.get_exposed_port(ch.port)),
+        database=ch.dbname,
+        user=ch.username,
+        password=ch.password,
+    )
+    client.command("""
+        CREATE TABLE orders (
+            o_orderkey       Int32,
+            o_custkey        Int32,
+            o_orderstatus    String,
+            o_totalprice     Decimal(15,2),
+            o_orderdate      Date,
+            o_orderpriority  String,
+            o_clerk          String,
+            o_shippriority   Int32,
+            o_comment        String
+        ) 
+        ENGINE = MergeTree
+        ORDER BY (o_orderkey)
+    """)
+    data_path = os.path.join(
+        os.path.dirname(__file__), "../../resource/tpch/data/orders.parquet"
+    )
+    client.insert_df("orders", pd.read_parquet(data_path))
+    request.addfinalizer(ch.stop)
+    return ch
+
+
+@pytest.mark.postgres
```

Comment:
Is it correct? I guess it should be clickhouse?
```suggestion
@pytest.mark.clickhouse
```